### ISSUE TITLE
[Bug]: Pings are not being properly shown

### ISSUE THREAD

**Issue by Im-Fran** [SwiftcordApp/Swiftcord#119]
### Describe the Bug

The ping is shown as raw message like `<@userid>`

### Reproducing the Bug

1. Ping someone
2. See that it's shown as raw text

### Version

0.5.1 (Build 15)

### Category

Message rendering

### Screenshots

Here is a ping from someone else to me, is not shown as `@Fran` but as raw text with my user id
![Screenshot 2022-10-15 at 13 31 26](https://user-images.githubusercontent.com/30329003/195997795-70566d69-5bc2-4d9b-93f4-3ef5a1ad638e.png)

**Comment by Im-Fran** [comment]
I could try to fix it, but I would need time to look at the code and understand on how the text is parsed. Also it might not be a good work cuz I just started learning SwiftUI xD

**Comment by Im-Fran** [comment]
But the project does have like a parser for the discord markdown, right? Isn't it possible to add another parser to that package? that also can parse pings?

**Comment by cryptoAlgorithm** [comment]
@Im-Fran nope, the md parsing is done by `Text` itself. it's a closed implementation with no way to extend it, unfortunately. Ahhh, apple thinking they know best again

**Comment by cryptoAlgorithm** [comment]
Do join the Discord server to reach me faster :)
